fast64_internal/oot/c_writer/oot_scene_bootup.py

```diff
@@ -0,0 +1,139 @@
+import os, re, bpy
+import string
+from bpy.utils import register_class, unregister_class
+from ...utility import PluginError, writeFile
+from ..oot_constants import ootEnumHeaderMenuComplete
+from typing import Callable, Iterable, Any, List
+
+
+def setBootupScene(exportPath: str, entranceIndex: str, options: Any):
```

Comment:
fyi for type annotation with types not yet defined, you can use strings:
```suggestion
def setBootupScene(exportPath: str, entranceIndex: str, options: "OOTBootupSceneOptions"):
```